ci/awsClient.sc

```diff
@@ -94,6 +94,8 @@ def uploadFileAndSha(uploadFile: Path, s3path: S3Path): Artifact = {
   val shaFile = fileUtil.writeSha1ForFile(uploadFile)
```

Comment:
How about using `createSha1OfFile` here and then reuse the sha for writing and echoing?